documentation/projects/proposals/ingestion_server_removal/20240328-implementation_plan_ingestion_server_removal.md

```diff
@@ -227,14 +226,11 @@ configuration we can ensure that the latest indexer-worker image is pulled each
 time, which means that any changes to the reindexing code will become live in
 production as soon as the new docker image is published after merging on `main`.
 
-The EC2 approach uses an ASG to achieve a similar result. Because the ASG will
+The EC2 approach uses EC2 operators to achieve a similar result. Because we will
```

Comment:
small question: does "EC2 operators" refer to https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/_api/airflow/providers/amazon/aws/operators/ec2/index.html?

Reply:
It does (there's also one use of an EC2 hook, also from the aws provider). I've linked to each of the specific operators/hooks I'm referring to in the implementation details below! All existing resources.

Reply:
You got to my comment too quickly :laughing: I was just going to delete it after seeing the docs. Thanks!